katharsis-jpa/src/main/java/io/katharsis/jpa/internal/query/JoinRegistry.java

```diff
@@ -0,0 +1,115 @@
+package io.katharsis.jpa.internal.query;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.persistence.criteria.From;
+import javax.persistence.criteria.JoinType;
+
+import io.katharsis.jpa.internal.meta.MetaAttribute;
+import io.katharsis.jpa.internal.meta.MetaAttributePath;
+import io.katharsis.jpa.internal.meta.MetaMapAttribute;
+import io.katharsis.jpa.internal.query.backend.JpaQueryBackend;
+import io.katharsis.jpa.query.criteria.JpaCriteriaExpressionFactory;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'io.katharsis.jpa.query.criteria.JpaCriteriaExpressionFactory'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)